Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -383,7 +384,7 @@ namespace pixelTopology {
     static constexpr uint32_t maxCellTracks = 48;
     static constexpr uint32_t maxHitsOnTrack = 10;
     static constexpr uint32_t maxHitsOnTrackForFullFit = 6;
-    static constexpr uint32_t avgHitsPerTrack = 4;
+    static constexpr uint32_t avgHitsPerTrack = 5;
```

Comment:
@AdrianoDee ?

Reply:
I think this is "technical" in the sense that it affects only how much memory is allocated in the `caStructures::HitToTupleT<TrackerTraits>`.

Reply:
Yes, indeed. While working on this I noticed we were over allocating using `maxHitsOnTrack*maxNumberOfTuples` for the hit histogram storage (in `TrackSoA` `HitContainer = cms::cuda::OneToManyAssoc<hindex_type, S + 1, H * S>;` ) so I went to `avgHitsPerTrack* maxNumberOfTuples` that seemed more reasonable but I wanted to be a bit more conservative.

Reply:
But I can revert it and address it somewhere else.

Reply:
I'm fine either way: make the change here, or make it in a dedicated PR.

Reply:
At this point I'm fine with either way as well. (but in general it would make the review of big re-engineering PRs easier if not-strictly-needed changes would be done elsewhere)